cfme/tests/configure/test_paginator.py

```diff
@@ -129,7 +135,9 @@ def test_paginator_config_pages(appliance, place_info):
                               for set_type in details_pages])
 def test_paginator_details_page(appliance, place_info, schedule):
     """
-        Check paginator is visible for access control pages + schedules
+        Check paginator is visible for access control pages + schedules.
+        If paginator is present, check that all options are present in items per page.
+        https://bugzilla.redhat.com/show_bug.cgi?id=1515952
```

Comment:
@mshriver and I just had a dicussion on this, this should be under the `Bugzilla` key and just the id. So:

```
Bugzilla:
    151952
```